core/scripts/cre/environment/configs/single-don.toml

```diff
@@ -34,6 +34,10 @@
 # uncomment as needed and adjust paths to enable these capabilities and have them copied to containers/pods
 # these default paths assume that binaries are placed in core/scripts/cre/environment/ folder
 # cron_capability_binary_path = "./cron"
+cron_capability_binary_path = "/Users/lautarofernandez/Desktop/cronBinary/cron"
+#evm_capability_binary_path = "/Users/lautarofernandez/Desktop/cronBinary/evm"
+evm_capability_binary_path = "/Users/lautarofernandez/repos/capabilities/chain_capabilities/evm/evm"
```

Comment:
we don't want to commit these path, they should stay as `./<capability_name>`

Reply:
indeed, this won't be uploaded in the final review.